ccd-definition/CaseEventToFields/otherProposalDecision.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+  "LiveFrom": "01/01/2017",
+  "CaseTypeID": "CARE_SUPERVISION_EPO",
+  "CaseEventID": "otherProposalDecision",
+  "CaseFieldID": "allocationDecision_Label",
+  "PageFieldDisplayOrder": 1,
+  "DisplayContext": "READONLY",
+  "PageID": 1,
+  "PageDisplayOrder": 1,
+  "PageColumnNumber": 1,
+  "FieldShowCondition": "allocationProposal!=\"\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "otherProposalDecision",
+    "CaseFieldID": "missingProposal_Label",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "allocationProposal!=\"*\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
this will always appear when the allocation missing label is shown, unless there is something I am missing?

how can we display "It should be a ${allocationProposal.proposal} as there are previous proceedings which were heard by a ${allocationProposal.proposal}..." if there is no allocationProposal?

Reply:
yes we don't want to display "It should be a ${allocationProposal.proposal} as there are previous proceedings which were heard by a ${allocationProposal.proposal}..." as it displays the syntax when there is no allocationProposal. So for when there is no allocationProposal we want to display another label, "The local authority has not made an allocation proposal."

Reply:
this functionality is now changed